packages/types/src/Locale.ts

```diff
@@ -4,6 +4,7 @@ enum Locale {
   EN_GB = 'en-GB',
   EN_US = 'en-US',
   ES_ES = 'es-ES',
+  FA_FA = 'fa-Fa',
```

Comment:
Those are IETF language tags, so for Persian used in Iran the code should be `fa-IR`

```suggestion
  FA_IR = 'fa-IR',
```